src/System.Security.Cryptography.RandomNumberGenerator/tests/RandomNumberGeneratorTests.cs

```diff
@@ -202,8 +202,8 @@ private static void AssertNeutralParity(byte[] random)
 
             // Over the long run there should be about as many 1s as 0s.
             // This isn't a guarantee, just a statistical observation.
-            // Allow a 3% tolerance band before considering it to have gotten out of hand.
-            Assert.True(bitDifference < 0.03);
+            // Allow a 7% tolerance band before considering it to have gotten out of hand.
+            Assert.True(bitDifference < 0.07);
```

Comment:
Nit: print out the value of `bitDifference` in the assert message so if it fails ever again we know whether it's way off or just 0.071